src/tests/dictionaryConcepts/components/InternalMapping.test.js

```diff
@@ -18,6 +26,24 @@ describe('render MappingModal', () => {
     expect(wrapper.length).toEqual(1);
   });
 
+  it('should render without breaking', () => {
```

Comment:
What should render without breaking?

Reply:
Thanks for the keen eye, I will make it more descriptive.